am-lib/src/main/java/uk/gov/hmcts/reform/amlib/internal/models/Role.java

```diff
@@ -15,4 +17,5 @@
     private final RoleType roleType;
     private final SecurityClassification securityClassification;
     private final AccessType accessType;
+    private LocalDateTime lastUpdate;
```

Comment:
I'm not sure using LocalDateTime for timestamps is suitable for auditing. LocalDateTime omits the timezone, which means that it doesn't capture the actual moment that the transaction happened. It would be better to use java.time.Instant which captures the moment in UTC

Reply:
We can use instant..But I think we are just using one time zone.out of one timezone it won't get used right?